guides/common/modules/con_boot-disk-workflow.adoc

```diff
@@ -0,0 +1,17 @@
+[id="boot-disk-workflow"]
+= Boot disk workflow
+
+Using a boot disk to provision a host involves the following steps:
+
+. Generate a boot disk ISO image.
+. Write the image to an external storage device, for example by using the `dd` utility or `livecd-tools`.
```

Comment:
In some cases you don't have to do that. For example, if I want to spin up a virtual machine with Libvirt, I'll just simply point to the iso file and Libvirt will take care of it, for example:
```
virt-install  --name=test.example.com \                                              
              --vcpus=4 \
              --memory=4000 \
              --disk size=20 \
              --os-variant=centos-stream9 \
              --network "network=foreman_default,mac=00:aa:bb:17:17:01" \
              --connect qemu:///system \
              --boot cdrom,hd \
              --cdrom=/path/to/image.iso
```

Reply:
So what do you suggest? Should I make this step optional?